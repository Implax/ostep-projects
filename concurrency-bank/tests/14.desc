Correctness test: multiple negative transactions (wife)
