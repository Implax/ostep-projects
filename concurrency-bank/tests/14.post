rm Husband.txt && rm Wife.txt