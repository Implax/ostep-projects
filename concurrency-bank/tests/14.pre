ln -sf $(pwd)/tests/assets/no_transactions_wel.txt /tmp/Husband.txt && ln -sf $(pwd)/tests/assets/negative_transactions.txt /tmp/Wife.txt
