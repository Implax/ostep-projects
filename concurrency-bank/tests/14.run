./bank 400 tests/assets/no_transactions_wel.txt tests/assets/negative_transactions.txt
