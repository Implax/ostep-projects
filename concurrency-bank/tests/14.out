Opening balance: 400
Withdraw: -600, User: Wife, Transaction declined
Withdraw: -600, User: Wife, Transaction declined
Withdraw: -600, User: Wife, Transaction declined
Withdraw: -600, User: Wife, Transaction declined
Closing balance: 400
